independent-publisher-connectors/AzureADApplications/apiProperties.json

```diff
@@ -0,0 +1,62 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "",

```

Comment:
@paulculmsee We have a practice to keep the client id value as `"[[DUMMY]]"`. Can you please update it accordingly?